spec/support/user4.rb

```diff
@@ -0,0 +1,8 @@
+class User4 < ActiveRecord::Base
```

Comment:
I started this trend but now that we have more variants of user in the tests, perhaps we should add a comment at the top explaining exactly what each one is testing?